debs/bionic/archivematica/debian-dashboard/postinst

```diff
@@ -11,9 +11,6 @@ chmod -R g+s $logdir
 echo "Removing Django's .pyo files to aid upgrade"
 find /usr/local/lib/python2.7/dist-packages/django/ -name '*.pyo' -delete
 
-# Install & upgrade dashboard requirements
-pip install --upgrade -r /usr/share/archivematica/dashboard/requirements.txt
```

Comment:
Is anyone able to explain me why we need this? If it's unneeded, I'm proposing to get rid of it.